app/controllers/sessions_controller.rb

```diff
@@ -21,6 +21,17 @@ def create
     end
   end
 
+  def destroy
+    if session[:current_user_id]
+      session.delete(:current_user_id)
+      redirect_to root_path, notice: "You've successfully logged out!"
+    else
+      # rubocop:disable I18n/RailsI18n/DecorateString
```

Comment:
This is a best solution, but this cop i agree to disable globaly, in rubocop.yml